src/components/forms/PhoneInput.tsx

```diff
@@ -133,6 +133,15 @@ export const PhoneInput = (props: PhoneInputProps) => {
     [setNumber, currentCountry, onChange]
   )
 
+  const maxLength = React.useMemo(() => {
+    const { maxNumberLength, countryCode } = currentCountry
+    if (isInternational) {
+      return maxNumberLength
+    } else {
+      return maxNumberLength + (countryCode === CountryCode.AU ? 2 : 1)
```

Comment:
only thought would be to move this into a prop on the country so there are 2 max lengths set. Intl version and local version, and then you read out the correct one based on the `isInternational` flag.

Reply:
@sumac13 this PR isn't the fix - Jess is working on the fix separately